notification-service/build.gradle

```diff
@@ -21,14 +21,15 @@ configurations {
 }
 
 repositories {
-	mavenCentral()
 	maven {
-        credentials {
-            username "$gemfireRepoUsername"
-            password "$gemfireRepoPassword"
-        }
-        url = uri("https://commercial-repo.pivotal.io/data3/gemfire-release-repo/gemfire")
-    }
+		if (project.hasProperty('gemfireRepoUsername')) {
```

Comment:
Those defined in the root + specific one for gemfire. Only set creds if they are set in property. If we dont do this we should instead complain w/ a particular reason (ie "You need to set your gemfire creds..." etc).